packages/web3-core/src/web3_context.ts

```diff
@@ -78,6 +79,7 @@ export class Web3Context<
 	private _subscriptionManager?: Web3SubscriptionManager<API, RegisteredSubs>;
 	private _accountProvider?: Web3AccountProvider<Web3BaseWalletAccount>;
 	private _wallet?: Web3BaseWallet<Web3BaseWalletAccount>;
+	private static readonly _batchRequest = Web3BatchRequest;
```

Comment:
A static but private? Can we not simply use the `Web3BatchRequest` directly?

Reply:
I changed it and exposed Web3BatchRequest directly